### ISSUE TITLE
Events for trees in navigation not triggered

### ISSUE THREAD

**Issue by NePheus** [almasaeed2010/AdminLTE#1536]
The collpase and expand events for the tree navigation items are never triggered. You want to trigger it on the ul elements of the depending tree, but its always undefined. See:
https://github.com/almasaeed2010/AdminLTE/blob/master/dist/js/adminlte.js#L457
https://github.com/almasaeed2010/AdminLTE/blob/master/dist/js/adminlte.js#L468
this.element is always undefined because 'this' is the reference to the slideDown/Up event. You have to save it to a variable before the animate call, i.e.:
```
var $element = $(this.element);
tree.slideDown(this.options.animationSpeed, function () {
      $element.trigger(expandedEvent)
})
```
Then the event gets triggered as it should.

**Comment by almasaeed2010** [comment]
@NePheus good catch! Thanks for reporting. Will fix soon.

**Comment by almasaeed2010** [comment]
Fixed in master (v2.4)
